docs/guides/cache/azblob.md

```diff
@@ -27,20 +27,18 @@ $ docker buildx build . --push -t <registry>/<image> \
   --cache-from type=azblob,name=<cache-image>[,parameters...]
 ```
 
-Common parameters:
-
-- `name`: the name of the cache image.
-- `account_url`: the base address of the blob storage account, for example:
-  `https://myaccount.blob.core.windows.net`. See
-  [authentication](#authentication).
-- `secret_access_key`: specifies the
-  [Azure Blob Storage account key](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-keys-manage),
-  see [authentication](#authentication).
-
-Parameters for `--cache-to`:
-
-- `mode`: specify cache layers to export (default: `min`), see
-  [cache mode](./index.md#cache-mode)
+The following table describes the available CSV parameters that you can pass to
+`--cache-to` and `--cache-from`.
+
+| Parameter           | Option     | Value      | Default | Description                                        |
+| ------------------- | ---------- | ---------- | ------- | -------------------------------------------------- |
+| `name`              | Both       | string     |         | Required. The name of the cache image.             |
+| `account_url`       | Both       | string     |         | Base URL of the storage account.                   |
+| `secret_access_key` | Both       | string     |         | Blob storage account key, see [authentication][1]. |
+| `mode`              | `cache-to` | `min\|max` | `min`   | Cache layers to export, see [cache mode][2].       |
+
+[1]: #authentication
+[2]: ../#cache-mode
```

Comment:
This is a broken link when reading the markdown file, but I am pretty sure the other syntax (`./index.md#cache-mode`) breaks on the rendered docs site?

Reply:
or maybe there is an issue with reverse footnotes with Jekyll :thinking:

Reply:
I was misremembering that mixing file-based references with HTML IDs would somehow cause broken links... But I tested it now and it works with Jekyll. Might've been something from Hugo. I will fix this link.